tests/common/include/tests/common/material_interaction.inl

```diff
@@ -322,6 +322,26 @@ TEST(material_interaction, telescope_geometry_energy_loss) {
     EXPECT_NEAR(new_var_qop, dvar_qop, 1e-10);
 
     // @todo: Validate the backward direction case as well?
+    // Flip the stepper direction and momentum for backward filtering
+    state._stepping().flip();
```

Comment:
I am wondering: Why do we flip the momentum of the actual track state? Does it not work to leave the track state untouched and instead flip the sign of the step-size as we do during overstepping? The Runge-Kutta algorithm at least should work in both directions

Reply:
I asked the same question in other PR: The navigation would be OK if we don't flip the momentum?

Reply:
I removed the backward navigation part